### PR TITLE
Added Geni provider

### PR DESCRIPTION
**Pull request by papinto**
I have a working provider for Geni.com that I'd like to contribute.

### CODE REVIEW

**Comment by TerribleDev** [comment]
Thank you very much for the contribution!! I'll look this over wednesday-ish and get back to you.

File: src/Owin.Security.Providers.Geni/GeniAuthenticationOptions.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        public ISecureDataFormat<AuthenticationProperties> StateDataFormat { get; set; }
+
+        /// <summary>
+        ///     Initializes a new <see cref="GeniAuthenticationOptions" />
+        /// </summary>
+        public GeniAuthenticationOptions()
+            : base("Geni")
+        {
+			Caption = Constants.DefaultAuthenticationType;
+            CallbackPath = new PathString("/signin-Geni");
```
Comment (TerribleDev) [inline src/Owin.Security.Providers.Geni/GeniAuthenticationOptions.cs]: can you make the default callback path `/signin-geni` all lower case?

Comment (papinto) [inline src/Owin.Security.Providers.Geni/GeniAuthenticationOptions.cs]: Sure.
Is there a way to add to this pull request or should I make a new one?

File: src/Owin.Security.Providers.Geni/GeniAuthenticationOptions.cs [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        /// </summary>
+        /// <value>
+        ///     The back channel timeout in milliseconds.
+        /// </value>
+        public TimeSpan BackchannelTimeout { get; set; }
+
+        /// <summary>
+        ///     The request path within the application's base path where the user-agent will be returned.
+        ///     The middleware will process this request when it arrives.
+        ///     Default value is "/signin-Geni".
```
Comment (TerribleDev) [inline src/Owin.Security.Providers.Geni/GeniAuthenticationOptions.cs]: can you fix this comment?

**Pull request merged**